_posts/2020-02-04-mirror_mirror.markdown

```diff
@@ -152,14 +155,14 @@ BORING
 ```haskell
 instance ToJSON User where
     toJSON user = object
-        [ "userName" =. userName user
-        , "userAge" =. userAge user
-        , "userFavorateAnimal" =. userFavoriteAnimal user
+        [ "userName" .= userName user
```

Comment:
Perhaps you mistook the =. from persistent with .= from aeson?
https://hackage.haskell.org/package/aeson-1.5.1.0/docs/Data-Aeson.html#v:.-61-